Practical-Forms/Borgification Volunteer Form.txt

```diff
@@ -13,7 +13,7 @@ This form is to signify that a member of the crew wishes to be made into a cybor
 		[bullet/] Memory loss
 		[bullet/] Discomfort in the Brain or similar areas
 		[bullet/] Possible tampering with personality by parties hostile to Nanotransen©
-		[bullet/] Possible tampering with personality by Clowns and similar personel
+		[bullet/] Possible tampering with personality by Clowns and similar personal

```

Comment:
Most likely intended to be "[personnel](https://www.merriam-webster.com/dictionary/personnel)," not "personal."